webapp/src/client/client.js

```diff
@@ -1,8 +1,11 @@
-import request from 'superagent';
+import {Client4} from 'mattermost-redux/client';
+import {ClientError} from 'mattermost-redux/client/client4';
 import AuthenticationContext from 'adal-angular';
 
+
 import {isDesktopApp} from '../utils/user_utils';
 import {Periods} from '../constants';
+import {id as pluginID} from `../manifest`
```

Comment:
Build is failing due to using backticks here

```suggestion
import {id as pluginID} from '../manifest';
```